priprava/lekce3/7.py

```diff
@@ -0,0 +1,31 @@
+from pprint import pprint
+# 7.a) Napište program, který vypíše pro první den kolik pasažérů jelo celkem směrem tam a kolik směrem zpět.
+
+pasazeri = open("pasazeri.txt")
+passengers = [p for p in pasazeri]
+pprint(passengers)
+
+passengers = [p.strip() for p in passengers]
+pprint(passengers)
+
+passengers = [p.split(" ") for p in passengers]
+pprint(passengers)
+
+dny = [[p.split(",") for p in passengers[x]] for x in range(len(passengers))]
```

Comment:
Nechces misto `range` chroustat radsi primo hodnoty `passengers`? Mne ty indexy prijdou vzdycky matouci, je to jako jeste jedna vrstva abstrakce navic.